compiler/test/Transforms/boundedShapeInference.mlir

```diff
@@ -38,14 +38,6 @@ func.func @registered_shape_infer(%arg0 : tensor<?x4xf32> {byteir.bounded_shape
 //CHECK-NEXT: %0 = mhlo.custom_call @byteir.non_zero(%arg0) : (tensor<?x4xf32, {byteir.bounded_shape = [8, 4]}>) -> tensor<?xi64, {byteir.bounded_shape = [32]}>
 //CHECK-NEXT: return %0 : tensor<?xi64, {byteir.bounded_shape = [32]}>
```

Comment:
seems need to change this type to `tensor<?x2xi64, {byteir.bounded_shape = [32, 2]}>`

Reply:
Yes, I accidentally left it out